mono/tests/Makefile.am

```diff
@@ -2866,7 +2864,9 @@ TESTS_UNHANDLED_EXCEPTION_255_SRC =	\
 	unhandled-exception-5.cs	\
 	unhandled-exception-6.cs	\
 	unhandled-exception-7.cs	\
-	unhandled-exception-8.cs
+	unhandled-exception-8.cs	\
+	threadpool-exceptions2.cs	\
```

Comment:
This means the `threadpool-exceptions2.cs` and `threadpool-exceptions5.cs` tests are expected to have exit code = 255? That sounds wrong.

Reply:
Yes, that's expected. They are testing the unhandled process exit caused by user exception